float32 value
---
float32 oldValue
float32 newValue